azext_iot/common/shared.py

```diff
@@ -17,36 +26,80 @@ class SdkType(Enum):
     dps_sdk = 5
 
 
+# pylint: disable=too-few-public-methods
 class EntityStatusType(Enum):
+    """
+    Resource status.
+    """
     disabled = 'disabled'
     enabled = 'enabled'
 
 
+# pylint: disable=too-few-public-methods
 class SettleType(Enum):
+    """
+    Result type of operation.
+    """
     complete = 'complete'
     abandon = 'abandon'
     reject = 'reject'
 
 
+# pylint: disable=too-few-public-methods
 class DeviceAuthType(Enum):
+    """
+    Device Authorization type.
+    """
     shared_private_key = 'shared_private_key'
     x509_thumbprint = 'x509_thumbprint'
     x509_ca = 'x509_ca'
 
 
+# pylint: disable=too-few-public-methods
 class KeyType(Enum):
+    """
+    Shared private key
+    """
     primary = 'primary'
     secondary = 'secondary'
 
 
+# pylint: disable=too-few-public-methods
 class AttestationType(Enum):
+    """
+    Type of atestation (TMP or certificate based)
```

Comment:
Looking for uniformity on periods '.' at the end of these comment blocks. I don't have a strong opinion on whether or not it should be there, just the same should apply across the board.

Reply:
GAH!  I missed one.  Fixed in commit: https://github.com/Azure/azure-iot-cli-extension/pull/4/commits/087bb44c55bb60ddbbac8fe1917adf42827a6a2b